darktrace.json

```diff
@@ -0,0 +1,1193 @@
+                    "data_type": "string"
+                },
+                {
+                    "column_name": "Destination Port",
+                    "column_order": 5,
+                    "data_path": "action_result.data.*.dest_port",
+                    "data_type": "numeric"
+                },
+                {
+                    "column_name": "Protocol",
+                    "column_order": 1,
+                    "data_path": "action_result.data.*.proto",
+                    "data_type": "string"
+                },
+                {
+                    "column_name": "Source Hostname",
+                    "column_order": 4,
+                    "data_path": "action_result.data.*.src_hostname",
+                    "data_type": "string"
+                },
+                {
+                    "column_name": "Source IP",
+                    "column_order": 3,
+                    "data_path": "action_result.data.*.src_ip",
+                    "data_type": "string"
+                },
+                {
+                    "column_name": "Source Port",
+                    "column_order": 2,
+                    "data_path": "action_result.data.*.src_port",
+                    "data_type": "numeric"
+                },
+                {
+                    "column_name": "Timestamp",
+                    "column_order": 0,
+                    "data_path": "action_result.data.*.time",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "parameters": {
+                "model_breach_id": {
+                    "contains": [
+                        "darktrace model breach id"
+                    ],
+                    "data_type": "string",
+                    "order": 0,
+                    "description": "See artifact details to get the model_breach_id",
+                    "primary": true,
+                    "required": true
+                }
+            },
+            "read_only": true,
+            "render": {
+                "type": "table"
+            },
+            "type": "investigate",
+            "versions": "EQ(*)"
+        }
+    ],
+    "app_version": "1.0.2",
```

Comment:
Would it be possible to re-order the .json file to have the app info at the top? I don't think it's strictly necessary, but would be nice. Specifically, move everything from line 1056 and below to the top. i.e.

- App info
- Asset config
- PIP wheels
- Action definitions, etc.